### ISSUE TITLE
Purge luckPerms

### ISSUE THREAD

**Issue by danyadanya2005** [AuthMe/AuthMeReloaded#1947]
I apologize for my bad English, I translate through Google)
Dear plugin author, please add for versions 1.14.4+ a function, parameter for luckPerms plugin
you have
Purge:
     # If enabled, AuthMe automatically purges old, unused accounts
     useAutoPurge: true
     # Do we need to remove permissions?
     removePermissions: true
many probably would like to replace permissions with luckPerms (permissions are not updated)
  example
     # Do we need to remove luckPerms?
     removeluckPerms: true

it is necessary that players who do not enter the server for a long time remove groups from luckPerms

**Comment by ljacqu** [comment]
Would it be enough if we added an "onPurge" section in the commands.yml? Does LuckPerms have a command to achieve what you want?

**Comment by danyadanya2005** [comment]
f LuckPerms had what I needed, I would not write to you. no there. and I asked them, they answered no. As for the Purge item, it does not work at all now. just cleans authme. I installed the PEXPlayersRemover plugin; it cleans the permissions. Essentials and playerdata in the world folder are not cleaned. if you can add worldguard cleanup. worldguard wrote in a discord they refused to add an item to the config to clear old players если у LuckPerms было то что мне надо я бы не писал вам. нету там. и я у них спрашивал , ответили нету. что касается пункта Purge сейчас он не работает вообще . только чистит authme. я поставил плагин PEXPlayersRemover он чистит пермишенс. ессентиалс и playerdata в папке world не чистится. если можете добавте очистку worldguard. worldguard писал в дискорде они отказались добавлять пункт в конфиг очистку старых игроков08.12.2019, 22:30, "ljacqu" <notifications@github.com>:Would it be enough if we added an "onPurge" section in the commands.yml? Does LuckPerms have a command to achieve what you want?

**Comment by danyadanya2005** [comment]
> Would it be enough if we added an "onPurge" section in the commands.yml? Does LuckPerms have a command to achieve what you want?

no, I'm wrong. groups in permissions plugin cleans but leaves players in permissions
example:
  0de6eaf5-cb46-35ae-8883-4509d230b5cc:
    group:
    - premium
    options:
      name: NBiryuchela
      prefix: '&7&l[&3&lШалунья,Ох_Шалунья&7&l]&6&l'

after cleaning
  0de6eaf5-cb46-35ae-8883-4509d230b5cc:
    group: []
    options:
      name: NBiryuchela
      prefix: '&7&l[&3&lШалунья,Ох_Шалунья&7&l]&6&l'

your plugin should remove the idd prefixes of players, delete the old regions of WorldGuard players

**Comment by dmitriJ2101** [comment]
> Would it be enough if we added an "onPurge" section in the commands.yml? Does LuckPerms have a command to achieve what you want?

Add the onPurge parameter to commands.yml
